### ISSUE TITLE
Disable or delete add-ons not compatible with Firefox or Firefox for Android

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#8476]
In https://github.com/mozilla/addons-server/issues/8344 we want to reject submissions of add-ons not compatible with Firefox or Firefox for Android. We also need to take care of the existing ones, disabling or deleting them.

Among these, those compatible with Thunderbird or Seamonkey will show up on addons.thunderbird.net, which will run a copy of our database, having done the same thing in reverse (removing add-ons not compatible with Thunderbird or Seamonkey).

**Comment by diox** [comment]
Could almost be a migration but we probably want more control over when to fire it, so it needs to be a management command.

It's essentially just a matter of doing:
`Addon.objects.public().exclude(appsupport__app__in=(amo.FIREFOX.id, amo.ANDROID.id)).delete()`

(The question is, do we want to delete them permanently, soft-delete them, or just disable them)

**Comment by eviljeff** [comment]
IMO, soft-delete for now.  We can actually delete later once we know everything is okay (perhaps with the legacy bonfire of the addons).

**Comment by diox** [comment]
Another query we'll want to run is `ApplicationsVersions.objects.filter(application__in=(amo.THUNDERBIRD.id, amo.SEAMONKEY.id))`. With the patch for https://github.com/mozilla/addons-server/issues/8344 active, that will ensure old compatibility information is removed (which will matter since those applications will be invalid in the form used to edit compatibility info in devhub)

**Comment by AlexandraMoga** [comment]
I have done a parallel search for add-ons compatible with Thunderbird and Seamonkey but _incompatible_ with Firefox and FF for Android and checked if they were listed on AMO (addons-dev.allizom.org):
Results:
1. Thunderbird and Seamonkey add-ons are listed among search suggestions on AMO (filed #8645)
2. Thunderbird and Seamonkey add-ons are hosted on addons.thunderbird.net. For example, if I open an add-on details page on AMO and I'm adding in the URL an add-on slug specific for Thunderbird, I'm immediately redirected to `addons.thunderbird.net/thunderbird/addon/{addon_slug}`.
3. Thunderbird and Seamonkey compatibility is not hidden on the version info page for older add-ons - see https://addons-dev.allizom.org/en-US/firefox/addon/reminderfox/versions/

@diox please let me know if this method is correct or if there are other areas to be covered. Also, let me know what you think about point 3.

**Comment by diox** [comment]
Sorry, this is a bit early to test this as I added the command to delete the add-ons but haven't run it yet. I'll get back to you when I've done that (it's not urgent as we're unlikely to need it this week anyway)

**Comment by ioanarusiczki** [comment]
I verified this issue on AMO dev with FF61(Win10).

- Opening an add-on detail page on AMO  and replacing the URL ( thunderbird/addon/purge/?src=hp-dl-upandcoming ) will redirect to https://addons.thunderbird.net/en-US/ on the detail page of the searched add-on.  Same with a Seamonkey compatible add-on. 

![redirects to thunderbird2](https://user-images.githubusercontent.com/33448286/42496309-40b262e0-842e-11e8-9263-96693076bbc5.gif)

- I verified for a couple of add-ons the listed versions and didn't find on AMO any Thunderbird or Seamonkey references.

![sq lite](https://user-images.githubusercontent.com/33448286/42496555-1d567e7a-842f-11e8-838f-9e1048de40f4.gif)

![session manager](https://user-images.githubusercontent.com/33448286/42496566-251d813a-842f-11e8-88d3-ef3698af60ba.gif)
